java/src/main/native/src/csv_chunked_writer.hpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cassert>
+
+#include <cudf/io/csv.hpp>
+
+#include "jni_writer_data_sink.hpp"
+
+namespace cudf::jni::io {
+
+/**
+ * @brief Class to write multiple Tables into the jni_writer_data_sink.
+ *
+ * TODO: Consider moving to cpp/ in the future, if there is interest.
```

Comment:
Do we need a follow on issue for this todo?  Should we just remove it instead?

Reply:
Removed. No evidence of interest, currently.